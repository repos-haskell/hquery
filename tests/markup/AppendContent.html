<div id="foo">Hi, my name is</div>
