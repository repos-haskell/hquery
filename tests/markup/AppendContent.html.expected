<div id="foo">Hi, my name is Slim Shady</div>
